docs/pages/troubleshooting.md

```diff
@@ -0,0 +1,64 @@
+    
+It will display usage instructions. You will have to set the `RUN_SCANNER_HOME` to the path containing an unpacked version of the WAR.
+
+<a id="SequencerInvalid" />
+### Sequencing Output Directory 'Valid?' reads 'No'
+To access a sequencing output directory, Run Scanner needs to be provided a valid absolute path to the directory, and the tomcat user needs to have read and execute permissions on said directory. The processor in the configuration JSON must also be specified correctly, and be provided a valid time zone.
+
+For more information on the configuration JSON file, please refer to <a href="installation.html#setup">Setting Up Run Scanner</a>.
+
+<!-- TODO: These messages really could be more user-friendly in Run Scanner itself -->
+#### 'Path is null!'
+This message appears if Run Scanner is not provided a valid absolute path to a sequencing output directory. This message will also appear if no path is specified. Check your configuration JSON for a valid absolute path to the sequencing output directory.
+
+#### 'Path is not a directory!'
+This message appears if Run Scanner does not detect that the path specified is a directory. This may happen if the path leads to a file, or if Run Scanner does not have permission to access the directory. Check that the path in the configuration JSON leads to a directory for which the tomcat user has permission to read and execute permissions. Grant those permissions if necessary.
+
+#### 'Path cannot be read!'
+This message appears if the tomcat user does not have permission to read the sequencing output directory. Ensure read and execute permissions are granted to the tomcat user on the sequencing output directory and its contents.
+
+#### 'Path cannot be executed!'
+This message appears if the tomcat user does not have execute permission for the sequencing output directory. Ensure read and execute permissions are granted to the tomcat user on the sequencing output directory and its contents.
+
+#### 'Processor is null!'
+This message appears if Run Scanner cannot construct a Processor given the information specified in the configuration JSON. Ensure that the 'name', 'platformType', and 'parameters' fields are filled out appropriately as per <a href="installation.html#setup">Setting Up Run Scanner</a> and <a href="appendices.html#A">Appendix A: Processor Definitions</a>.
+
+#### 'TimeZone is null!'
+This message appears if the 'timeZone' field in the configuration JSON is blank or invalid. This field uses TZ Database names. Please refer to <a href="https://en.wikipedia.org/wiki/List_of_tz_database_time_zones#List">List of TZ Database names on Wikipedia</a>.
+
+<a id="Unreadable" />
+### Runs appear in Unreadable list
+#### Check that directory contains run data
+Directories will appear in the Unreadable list if they do not contain files which Run Scanner recognizes as run data. <!-- Vague --> Ensure that the directory contains run data.
```

Comment:
Directories will appear in the Unreadable list if they do not contain files which Run Scanner expects to find for a run. For example, for an Illumina run, this would happen if the `RunParameters.xml` was missing.